docs/developer/running/storage_operator.rst

```diff
@@ -0,0 +1,41 @@
+Running the storage operator locally
+====================================
+
+Requirements
+------------
+
+- `Go <https://golang.org/>`_ (1.12 or higher) and
+  `operator-sdk <https://github.com/operator-framework/operator-sdk>`_ (0.9 or
+  higher): to build the Kubernetes Operators
+- `Mercurial <https://www.mercurial-scm.org/>`_: some Go dependencies are
+  downloaded from Mercurial repositories.
+
+Prerequisites
+-------------
+
+- You should have a running Metalk8s cluster somewhere
+- You should have installed the dependencies locally with
+  ``cd storage-operator; go mod download``
+
+Procedure
+---------
+
+1. Copy the ``/etc/kubernetes/admin.conf`` from the bootstrap node of your
+   cluster onto your local machine
+
+2. Delete the already running storage operator, if any, with
+   ``kubectl --kubeconfig /etc/kubernetes/admin.conf -n kube-system
```

Comment:
This way we explicitly stay on the local machine as soon as we've finished grabbing the admin.conf from the remote bootstrap node.

```suggestion
   ``kubectl --kubeconfig <PATH_TO_LOCAL_ADMIN_CONF> -n kube-system
```

Reply:
Yeah but not everyone have `kubectl` installed locally, so I prefer to document the way from the bootstrap node.